storage/client_split_test.go

```diff
@@ -412,6 +412,64 @@ func TestStoreZoneUpdateAndRangeSplit(t *testing.T) {
 	}
 }
 
+// TestStoreRangeSplitWithMaxBytesUpdate tests a scenario where a new
+// zone config that updates the max bytes is set and triggers a range
+// split.
+//
+// TODO(kkaneda): Tweak this test if a new range can get a new value
+// of max bytes without another zone config change.
```

Comment:
yeah, we think it might be a bug. see discussion: https://github.com/cockroachdb/cockroach/pull/1337/files#r32207958